### PR TITLE
WEBEU3 - react-american-football-scoreboard - Abel Tedros

### PR DESCRIPTION
**Pull request by AbelTedros**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
 import BottomRow from "./BottomRow";
 
-function App() {
-  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+function App(props) {
+  //TODO: STEP 2 - Establish your application's state with some useState hooks.  You'll need one for the home score and another for the away score.
+
+    const [lionHomeScore, setlionHomeScore] = useState(0);
+
+    const [tigerHomeScore, settigerHomeScore] = useState(0);
```
Comment (OloruntobiAwoderu) [inline src/App.js]: Good naming conventions

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+
+    // for Lion's score: 
+    const resetLionScore = () => {
+        setlionHomeScore(0);
+  };
+
+  const incrementLionScore = () => {
+      setlionHomeScore(lionHomeScore +7);
+  };
+
```
Comment (OloruntobiAwoderu) [inline src/App.js]: Good use of variables to store your arrow functions

File: src/App.js (hunk 1 above)
Comment (AbelTedros) [reply src/App.js]: Thanks man :)

**Comment by AbelTedros** [comment]
> Good work on this project Abel!!!
> I'm really proud of you.
> You reached MVP and did well.
> Keep putting in the effort and keep up the good work

Thanks mate :) I'm slowly getting the hang of this 🥇

**Pull request merged**